corehq/apps/case_search/tests/test_filter_dsl.py

```diff
@@ -738,6 +738,14 @@ def _test_xpath_query(self, query_string, expected_filter, context=None):
         built_filter = build_filter_from_ast(parsed, context)
         self.checkQuery(built_filter, expected_filter, is_raw_query=True)
 
+    def test_match_all_error(self):
```

Comment:
Should this be renamed to `test_match_all` since it doesn't appear to test the error case